bitnami/ghost/values.yaml

```diff
@@ -400,9 +407,10 @@ ingress:
   ##   - Relay on Helm to create self-signed certificates by setting `ingress.selfSigned=true`
```

Comment:
Check the previous line (406). My suggestion:
```diff
-  ##   - Relay on cert-manager to create it by setting `ingress.certManager=true`
+  ##   - Relay on cert-manager to create it by setting the corresponding annotations
```

Reply:
Done in bacc9c7!